pkg/cloudprovider/azure.go

```diff
@@ -14,6 +14,10 @@ const (
 
 type azure struct{}
 
+func (a *azure) init(nodeList *v1.NodeList) error {
+	return nil
```

Comment:
```suggestion
	return fmt.Errorf("azure cloud provider not yet implemented")
```

Reply:
we actually have azure cloudprovider needed somewhere else. the provider is needed, but just nothing required in this init so far.